src/chrome/content/rules/rsbsrt.ru.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="rsbsrt.ru">
+	<target host="rsbsrt.ru" />
+	<target host="www.rsbsrt.ru" /> <!-- mismatch, wrong content -->
+
+	<rule from="^http://(www\.)?" to="https://" />
```

Comment:
Invalid regex: https://travis-ci.com/github/EFForg/https-everywhere/jobs/499842904#L332-L335

Reply:
>missing caret (^) in 'from' attribute: it should be "^http:" and not "http:"

Obviously wrong, the caret is present, it is obvious.

>missing trailing slashes in 'from' or 'to' when specifying full hostnames: 
it should be "https://eff.org/" and not "https://eff.org"

Not needed here.

The regex is valid and OK, it is the static analyser used flawed. Please don't call it invalid anymore, it is the static analyser that is invalid.